zipline/algorithm.py

```diff
@@ -98,10 +97,8 @@
 
 from zipline.utils.input_validation import ensure_upper_case, error_keywords
 from zipline.utils.cache import CachedObject, Expired
-from zipline.utils.calendars import (
-    default_nyse_schedule,
-    ExchangeTradingSchedule,
-)
+from zipline.utils.calendars.trading_calendar import default_nyse_calendar
```

Comment:
Feel free to merge without that change, but I do think it is an important API question regarding the new calendars, e.g. how should a consumer get a calendar instance.  I just don't have a sense of where it fits into the calendar plan.